tests/pos/expeimental-flag.scala

```diff
@@ -0,0 +1,18 @@
+//> using options -experimental -Yno-experimental
```

Comment:
`-Yno-experimental` is set to disable experimental features on this snapshot. `-experimental` is set to add `@experimental` to the top-level definitions.